src/palace/manager/api/overdrive/representation.py

```diff
@@ -486,6 +486,13 @@ def book_info_to_bibliographic(
                         quantity_measured=Measurement.AWARDS, value=str(num_awards)
                     )
                 )
+            if popularity := book.get("popularity"):
+                measurements.append(
+                    MeasurementData(
+                        quantity_measured=Measurement.POPULARITY,
+                        value=popularity,
+                    )
+                )
```

Comment:
This "popularity" measurement isn't new. I just moved it up here with the other remaining ("awards") measurement, so that all of the measurements are together.